NSS-Day-Cohort-68/rare-client-rare-team3
# Added functionality to create a comment

I've added support for a user to create a comment. This includes a button within PostDetails.js that will navigate to CommentForm.js, a save button on CommentForm.js that will POST comment info to database, and once POSTED, navigation to ViewComments.js.

Testing:
1. `git fetch`
2. `git checkout heidel/comments/create-a-comment/client`
3. As a logged in user, navigate to a post
4. Navigate to the post's details
5. Click "Add a Comment" <button>
6. enter in text as a Comment
7. Click Save <button>
8. confirm that comment data POSTed to database
9. confirm that you are then navigated to "Comments List"
